openwrt-mdnsd-cve-2020-11750/src/fuzzer.c

```diff
@@ -0,0 +1,369 @@
+/* This is a modified version of:
+ * https://git.openwrt.org/?p=project/mdnsd.git;a=blob_plain;f=dns.c;hb=45c4953b602962ae7ff335d9a346000f00680952
+ */
```

Comment:
I assume this is the GPL code. Just add to the comment the code is GPL'ed. Fine to add to the repo; there isn't any intellectual property here in the stand-alone binary created we'd be worried about.